### PR TITLE
Use GDAL as base image

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,13 +26,21 @@ RUN \
     apt-get install --assume-yes --no-install-recommends 'nodejs=14.*' && \
     apt-get clean && \
     rm --recursive --force /var/lib/apt/lists/*
+
+RUN \
+    . /etc/os-release && \
+    apt-get update && \
+    apt-get install --assume-yes --no-install-recommends
```
Comment (arnaud-morvan) [inline Dockerfile:33]: Here it install nothing but leave package cache in image ?

File: Dockerfile [hunk 2]
```diff
@@ -39,15 +50,18 @@ ENV TEST=false
 # Finally used for all misk task, will not be used on prod runtime
 
 FROM base AS tools
+SHELL ["/bin/bash", "-o", "pipefail", "-ceux"]
```
Comment (arnaud-morvan) [inline Dockerfile:53]: What does this ? Maybe add a comment.

**Review by arnaud-morvan — APPROVED** [review]
Looks good, just some things that I do not really understand.

File: Dockerfile (hunk 1 above)
Comment (sbrunner) [reply Dockerfile:33]: Oups thanks :-)

File: Dockerfile (hunk 2 above)
Comment (sbrunner) [reply Dockerfile:53]: => comment added:
# Fail on error on pipe, see: https://github.com/hadolint/hadolint/wiki/DL4006.
# Treat unset variables as an error when substituting.
# Print commands and their arguments as they are executed.

**Pull request merged**